docs/language/composite-types.mdx

```diff
@@ -1126,7 +1126,7 @@ resource R {}
 // Declare a constant for an array of resources.
 // Create two resources and move them into the array.
 //
-let resources <- [
+let resources: @[R] <- [
```

Comment:
The type annotation isn't necessary here, the type of the declaration can be inferred from the value.
In the other examples I used something like this below the declaration: 

```
// `resources` has type `@[R]`
```